pilot/pkg/networking/telemetry/telemetry_test.go

```diff
@@ -206,18 +220,33 @@ func TestBuildStatPrefix(t *testing.T) {
 			"%SERVICE_FQDN%",
 			"reviews.default.svc.cluster.local",
 			"v1",
+			0,
 			&model.Port{Name: "grpc-svc", Port: 7443, Protocol: "GRPC"},
 			&model.ServiceAttributes{
 				Name:      "reviews",
 				Namespace: "default",
 			},
 			"reviews.default.svc.cluster.local",
 		},
+		{
+			"Service With Port and Port Name pattern With Target Port",
+			"%SERVICE%_%SERVICE_PORT_NAME%_%SERVICE_PORT%",
```

Comment:
I think this is a breaking change in general. Not to mention that `SERVICE_PORT` has a well defined meaning that isn't "target port"?

If this is for customized names, would it be more appropriate to make a new `TARGET_PORT`?

This is only changing customized names, right? Not the default

Reply:
It can be viewed as a breaking change but for inbound using target_port makes more sense because cluster would be created with that and stats prefix would be inconsistent. We can use new TARGET_PORT to be on safe side.

Yes it is for customized names only

Reply:
I  have added TARGET_PORT - I will update Api docs once this is merged